src/pages/home/Home.tsx

```diff
@@ -1,88 +1,120 @@
+import data from "../../assets/data/data.json";
+import { HomeTopic } from '../../components/home_topic';
 
 const Home: React.FC = () => {
-
-  const moemorizedCases = useMemo(() =>  data.cases, []);
-  const moemorizedTopics = useMemo(() =>  data.topics, []);
+  const guides = data.topics;
 
   return (
     <IonPage>
-      <IonContent>
-        <IonListHeader>
-          Highlights
-        </IonListHeader>
-        <IonGrid>
-          <IonRow>
-            {moemorizedTopics.filter((o) => o.highlighted === true).map((highlight, index) => (
-              <IonCol size="12" size-sm="6" size-xl="4" key={index}>
-                <IonCard routerLink={"guides/" + highlight.id}>
-                  <IonCardHeader>
-                    <IonCardTitle>{highlight.title}</IonCardTitle>
-                  </IonCardHeader>
-                  <IonCardContent>{parse(highlight.snippet)}</IonCardContent>
-                </IonCard>
+      <IonContent className="ion-padding">
+        <h2>South Africa's Constitution</h2>
+        <IonSearchbar placeholder="Find guides, cases or sections..." onIonChange={(e) => (e)}></IonSearchbar>
+        <IonItem>
+          <IonGrid>
+            <IonRow>
+              <IonCol size="2">
+                <IonIcon size="large" icon={document} />
               </IonCol>
-            ))}
-          </IonRow>
-        </IonGrid>
-        <IonListHeader>
-          Popular Topics
-        </IonListHeader>
-        <IonGrid>
-          <IonRow>
-            {moemorizedTopics.filter((o) => o.featured === true).map((topic, index) => (
-              <IonCol size="12" size-sm="6" size-xl="4" key={index}>
-                <IonCard routerLink={"guides/" + topic.id}>
-                  <IonCardHeader>
-                    <IonCardTitle>{topic.title}</IonCardTitle>
-                  </IonCardHeader>
-                  <IonCardContent>
-                    <IonGrid>
-                      <IonRow>
-                        <IonCol size="3">
-                          <img src={"../../assets/images/" + topic.id + ".svg"} onError={(e) => { e.currentTarget.src = "../../assets/shapes.svg" }} alt={topic.title} />
-                        </IonCol>
-                        <IonCol size="9">
-                          {parse(topic.snippet)}
-                        </IonCol>
-                      </IonRow>
-                    </IonGrid>
-                  </IonCardContent>
-                </IonCard>
+              <IonCol size="7">
+                <h4>The Constitution</h4>
+              </IonCol>
+              <IonCol size="3">
+                <IonRouterLink class="ion-float-right" href="/search">Search</IonRouterLink>
               </IonCol>
+            </IonRow>
+            <p>Browse the Constitution for the provisions that you need.</p>
+            <IonButton href="/constitution" expand="block">Browse Sections</IonButton>
+          </IonGrid>
+        </IonItem>
+        <IonItem>
+          <IonGrid>
```

Comment:
Perhaps try using IonToolbar https://ionicframework.com/docs/api/toolbar here, rather than the grid. It has provision for an icon and for buttons -- maybe that'll get them to align nicely?